### ISSUE TITLE
more clearly label closed/superceded places in places.mapzen.com 

### ISSUE THREAD

**Issue by burritojustice** [whosonfirst/whosonfirst-www-api#118]
When searching for places, closed / superceded / ceased records show up, e.g. 

https://places.mapzen.com/search?q=mapzen&scope=names

Presume we include these results, we may want to consider how we sort and display records that are no longer current. Options include:

- sort order (list things that are current first, and list things that are not current last)
- have a checkbox that hides places that do not have currentness (maybe checked by default?)

On the page for a place that is not current, e.g. https://places.mapzen.com/id/907212541

- if it is closed, clearly label it as closed. (via `edtf:cessation`, `mz:is_current`? existentialism...)
- if it is `superseded`, clearly label it as `superseded`, and link to the place it is `superseded_by` (the spelunker has this above the map)

**Comment by dphiffer** [comment]
Yeah, this is currently way too subtle. I'll take a shot at this on my next pass.
